packages/ui-tree-browser/src/TreeBrowser/README.md

```diff
@@ -7,141 +7,428 @@ to receive a normalized data structure, examples can be seen at https://github.c
+            description={
+              <ScreenReaderContent>
+                TreeBrowser size selector
+              </ScreenReaderContent>
+            }
             variant="toggle"
-            onChange={this.handleSizeSelect}
+            onChange={handleSizeSelect}
           >
-            {this.sizes.map((size) => <RadioInput key={size} label={size} value={size} />)}
+            {sizes.map((size) => (
+              <RadioInput key={size} label={size} value={size} />
+            ))}
           </RadioInputGroup>
         </View>
 
         <TreeBrowser
-          size={this.state.size}
+          size={size}
           collections={{
             1: {
               id: 1,
-              name: "Assignments",
-              collections: [2,3],
+              name: 'Assignments',
+              collections: [2, 3],
               items: [3],
-              descriptor: "Class Assignments"
+              descriptor: 'Class Assignments'
+            },
+            2: {
+              id: 2,
+              name: 'English Assignments',
+              collections: [4],
+              items: []
+            },
+            3: {
+              id: 3,
+              name: 'Math Assignments',
+              collections: [5],
+              items: [1, 2]
             },
-            2: { id: 2, name: "English Assignments", collections: [4], items: [] },
-            3: { id: 3, name: "Math Assignments", collections: [5], items: [1,2] },
-            4: { id: 4, name: "Reading Assignments", collections: [], items: [4] },
-            5: { id: 5, name: "Advanced Math Assignments", items: [5]}
+            4: {
+              id: 4,
+              name: 'Reading Assignments',
+              collections: [],
+              items: [4]
+            },
+            5: { id: 5, name: 'Advanced Math Assignments', items: [5] }
           }}
           items={{
-            1: { id: 1, name: "Addition Worksheet" },
-            2: { id: 2, name: "Subtraction Worksheet" },
-            3: { id: 3, name: "General Questions" },
-            4: { id: 4, name: "Vogon Poetry" },
-            5: { id: 5, name: "Bistromath", descriptor: "Explain the Bistromathic Drive" }
+            1: { id: 1, name: 'Addition Worksheet' },
+            2: { id: 2, name: 'Subtraction Worksheet' },
+            3: { id: 3, name: 'General Questions' },
+            4: { id: 4, name: 'Vogon Poetry' },
+            5: {
+              id: 5,
+              name: 'Bistromath',
+              descriptor: 'Explain the Bistromathic Drive'
+            }
           }}
           defaultExpanded={[1, 3]}
           rootId={1}
         />
       </>
     )
   }
-}
 
-render(<Example/>)
+  render(<Example />)
+  ```
 
-```
+- ```js
+  const Example = () => {
+    const [size, setSize] = useState('medium')
+    const sizes = ['small', 'medium', 'large']
+
+    const handleSizeSelect = (e, size) => {
```

Comment:
I think this example is duplicated, there should be only two there, the class and the function based.

Reply:
Removed the duplicate